server/models/actionPlan.ts

```diff
@@ -12,13 +12,21 @@ export interface ActionPlanAppointment {
   sessionNumber: number
   appointmentTime: string | null
   durationInMinutes: number | null
+  appointmentDeliveryType: AppointmentDeliveryType | null
```

Comment:
isn't this non null on the backend model?

Reply:
It is nullable to be consistent with the fields `appointmentTime` and `durationInMinutes` also being nullable.

It needs to be nullable because when the user clicks on the edit appointment details page these specific fields will be null on the Appointment object but the Appointment itself will exist.